composer.json

```diff
@@ -86,6 +86,11 @@
             "Composer\\Config::disableProcessTimeout",
             "php matchbot-cli.php matchbot:push-donations"
         ],
+        "matchbot:reset": [
+            "Composer\\Config::disableProcessTimeout",
+            "php matchbot-cli.php matchbot:reset-matching",
+            "@doctrine:delete-and-recreate"
+        ],
```

Comment:
@NoelLH do we need to also do this? `redis redis-cli FLUSHALL`

Reply:
@tbg-jasonfung I don't think so. The whole purpose of `matchbot:reset-matching` is to remove just the relevant fund balance / matching data from Redis in a clean way, without unnecessarily nuking everything like caches that share the instance.